sdk/python/feast/field.py

```diff
@@ -46,6 +65,300 @@ def dtype(self) -> ValueType:
         """
         return self._dtype
 
+    @property
+    def presence(self) -> schema_pb2.FeaturePresence:
+        """
+        Getter for presence of this field
+        """
+        return self._presence
+
+    @presence.setter
+    def presence(self, presence: schema_pb2.FeaturePresence):
```

Comment:
Can you please add comments to the getters and setters. Specifically ones that actually explain what the getter and setter does. For example if we say `getter for shape` it adds no value. You can just copy the existing descriptions from TFDV

Reply:
Added